main.go

```diff
@@ -30,6 +31,7 @@ func main() {
 	cmd.Run = func(cmd *cobra.Command, args []string) {
 		internal.RunKubeStateMetricsWrapper(opts)
 	}
+	cmd.AddCommand(generate.GenerateCommand)
```

Comment:
Note: I have to add the subcommand here. Doing it inside the options package would result in cyclic imports.